lib/ret/hub.ex

```diff
@@ -62,6 +62,19 @@ defmodule Ret.Hub do
     |> HubSlug.unique_constraint()
   end
 
+  def changeset_for_new_name(%Hub{} = hub, attrs) do
```

Comment:
I already ran into this on my branch, but this also needs to regenerate the slug and you need to flip a setting on the slug module (see my PR)

Reply:
most likely you should just pull/merge the implementation of this bit from my branch